Firmware/cardreader.h

```diff
@@ -135,7 +135,7 @@ class CardReader
 
   bool autostart_stilltocheck; //the sd start is delayed, because otherwise the serial cannot answer fast enought to make contact with the hostsoftware.
   
-  int16_t nrFiles; //counter for the files in the current directory and recycled as position counter for getting the nrFiles'th name in the directory.
+  uint16_t nrFiles; //counter for the files in the current directory and recycled as position counter for getting the nrFiles'th name in the directory.
```

Comment:
are you absolutely sure that `nrFiles` is never abused with a negative value?

Reply:
Just looked again. Nothing uses it as a signed number. Only as a file count / file index (0 indexed)